js/constants.js

```diff
@@ -1,3 +1,87 @@
+		 M645.5,92.6h25.3c-0.3-4.8-1.6-8.3-3.9-10.7c-2.2-2.4-5.2-3.5-8.8-3.5c-3.5,0-6.3,1.2-8.6,3.6C647.3,84.3,646,87.9,645.5,92.6z"/>
+	<path class="st0" d="M731.3,117.6c-3.4,2.6-6.6,4.6-9.8,5.9c-3.1,1.3-6.7,2-10.6,2c-3.6,0-6.7-0.7-9.4-2.1s-4.8-3.3-6.3-5.7
+		c-1.5-2.4-2.2-5-2.2-7.9c0-3.8,1.2-7.1,3.6-9.8c2.4-2.7,5.7-4.5,10-5.4c0.9-0.2,3.1-0.7,6.6-1.4c3.5-0.7,6.5-1.4,9-2
+		s5.2-1.3,8.1-2.2c-0.2-3.7-0.9-6.4-2.2-8.1s-4-2.6-8.1-2.6c-3.5,0-6.2,0.5-8,1.5s-3.3,2.5-4.6,4.4c-1.3,2-2.2,3.3-2.7,3.9
+		S703,89,701.3,89c-1.6,0-2.9-0.5-4.1-1.5c-1.1-1-1.7-2.3-1.7-3.9c0-2.5,0.9-4.8,2.6-7.1c1.7-2.3,4.4-4.2,8.1-5.7
+		c3.7-1.5,8.3-2.2,13.7-2.2c6.1,0,10.9,0.7,14.5,2.2c3.5,1.4,6,3.7,7.4,6.9c1.4,3.1,2.2,7.3,2.2,12.5c0,3.3,0,6,0,8.3s0,4.8-0.1,7.6
+		c0,2.6,0.4,5.4,1.3,8.2c0.9,2.8,1.3,4.7,1.3,5.5c0,1.4-0.7,2.7-2,3.9c-1.3,1.2-2.9,1.8-4.6,1.8c-1.4,0-2.8-0.7-4.2-2
+		C734.3,122.1,732.8,120.1,731.3,117.6z M730.3,97.4c-2,0.7-5,1.5-8.9,2.4c-3.9,0.8-6.6,1.4-8.1,1.8s-2.9,1.2-4.3,2.3
+		c-1.4,1.1-2,2.7-2,4.8c0,2.1,0.8,3.9,2.4,5.4c1.6,1.5,3.7,2.2,6.3,2.2c2.8,0,5.3-0.6,7.6-1.8c2.3-1.2,4-2.8,5.1-4.7
+		c1.3-2.1,1.9-5.6,1.9-10.4V97.4z"/>
+	<path class="st0" d="M772,105.3v11.6c0,2.8-0.7,4.9-2,6.4c-1.3,1.4-3,2.1-5.1,2.1c-2,0-3.7-0.7-5-2.1s-1.9-3.5-1.9-6.3V78.1
+		c0-6.3,2.3-9.4,6.8-9.4c2.3,0,4,0.7,5,2.2s1.6,3.6,1.7,6.5c1.7-2.9,3.4-5,5.1-6.5c1.8-1.5,4.1-2.2,7-2.2s5.8,0.7,8.5,2.2
+		c2.8,1.5,4.1,3.4,4.1,5.8c0,1.7-0.6,3.1-1.8,4.2s-2.4,1.7-3.8,1.7c-0.5,0-1.7-0.3-3.7-0.9s-3.7-0.9-5.2-0.9c-2,0-3.7,0.5-5,1.6
+		s-2.3,2.7-3,4.8c-0.7,2.1-1.2,4.6-1.5,7.5C772.1,97.6,772,101.1,772,105.3z"/>
+	<path class="st0" d="M851.3,108.2c0,1.7-0.5,3.6-1.6,5.6c-1,2-2.6,3.9-4.7,5.6c-2.1,1.8-4.8,3.2-8,4.3c-3.2,1.1-6.9,1.6-10.9,1.6
+		c-8.6,0-15.3-2.5-20.2-7.5c-4.8-5-7.3-11.8-7.3-20.2c0-5.7,1.1-10.8,3.3-15.2c2.2-4.4,5.4-7.8,9.6-10.2s9.2-3.6,15-3.6
+		c3.6,0,6.9,0.5,9.9,1.6s5.6,2.4,7.7,4.1c2.1,1.7,3.7,3.4,4.8,5.3s1.7,3.7,1.7,5.3c0,1.7-0.6,3.1-1.9,4.2c-1.2,1.2-2.7,1.7-4.5,1.7
+		c-1.2,0-2.1-0.3-2.9-0.9c-0.8-0.6-1.6-1.6-2.6-2.9c-1.7-2.6-3.5-4.5-5.3-5.8c-1.9-1.3-4.2-1.9-7.1-1.9c-4.1,0-7.4,1.6-10,4.8
+		c-2.5,3.2-3.8,7.6-3.8,13.2c0,2.6,0.3,5,1,7.2c0.6,2.2,1.6,4.1,2.8,5.6s2.7,2.7,4.4,3.5s3.6,1.2,5.7,1.2c2.8,0,5.2-0.6,7.2-1.9
+		s3.8-3.3,5.3-5.9c0.9-1.6,1.8-2.8,2.8-3.7c1-0.9,2.2-1.3,3.6-1.3c1.7,0,3.1,0.6,4.2,1.9C850.7,105.3,851.3,106.7,851.3,108.2z"/>
+	<path class="st0" d="M874.5,56.5v20.7c1.8-2,3.5-3.7,5.2-4.9c1.7-1.2,3.5-2.1,5.6-2.7c2-0.6,4.2-0.9,6.6-0.9c3.5,0,6.7,0.7,9.4,2.2
+		s4.9,3.7,6.5,6.5c1,1.7,1.7,3.5,2,5.6s0.5,4.4,0.5,7.1v26.7c0,2.8-0.6,4.9-1.9,6.3c-1.3,1.4-3,2.1-5.1,2.1c-4.6,0-6.9-2.8-6.9-8.5
+		V93.4c0-4.5-0.7-7.9-2-10.3s-3.8-3.6-7.6-3.6c-2.5,0-4.7,0.7-6.7,2.1s-3.5,3.3-4.5,5.8c-0.7,2.1-1.1,5.8-1.1,11.1v18.4
+		c0,2.8-0.6,4.9-1.9,6.3c-1.2,1.4-3,2.2-5.2,2.2c-4.6,0-6.9-2.8-6.9-8.5V56.5c0-2.8,0.6-4.9,1.8-6.4c1.2-1.4,2.9-2.1,5.1-2.1
+		c2.2,0,3.9,0.7,5.2,2.1C873.8,51.6,874.5,53.7,874.5,56.5z"/>
+</g>
+</svg>
+`;
+export {LOGO_MAIN, NO_POSTER_URL};
```

Comment:
do not use this pattern if possible. 
usually we write `export const foo = "bar"`, export occurs right at the variable declaration